### PR TITLE
PubMatic to execute tracker pixel in iframe.

### PR DESCRIPTION
**Pull request by pm-harshad-mane**
## Type of change

- [x] Bugfix
- [ ] Feature
- [ ] New bidder adapter
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other

## Description of change
PubMatic impression tracker pixel needs to be executed in iframe to avoid multiple times execution.
So we have added a new function createTrackPixelIframeHtml in utils and have used the same in PubMatic adapter.

### CODE REVIEW

**Comment by pm-harshad-mane** [comment]
Hello @mkendall07 ,

Do you need anything from me for code-review ?

**Comment by mkendall07** [comment]
@pm-harshad-mane 
Can you explain the need for the iframe vs image embed? I don't see the difference here.

File: src/utils.js [hunk 1]
```diff
@@ -471,6 +471,19 @@ exports.createTrackPixelHtml = function (url) {
...
+ * Creates a snippet of Iframe HTML that retrieves the specified `url`
+ * @param  {string} url plain URL to be requested
+ * @return {string}     HTML snippet that contains the iframe src = set to `url`
+ */
+exports.createTrackPixelIframeHtml = function (url) {
+  if (!url) {
+    return '';
+  }
+
+  return '<iframe frameborder="0" allowtransparency="true" marginheight="0" marginwidth="0" width="0" hspace="0" vspace="0" height="0" style="height:0p;width:0p;display:none;" scrolling="no" src="' + encodeURI(url) + '"></iframe>';
```
Comment (mkendall07) [inline src/utils.js]: Please use string template literals here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

**Comment by pm-harshad-mane** [comment]
Hello @mkendall07 ,

Agree with you that there is no much difference in executing a pixel in an image tag or an iframe tag.
But in our past-experience we have observed some discrepancies in case when our tracker call is executed in image tag as image tag with zero byte content reloads in some cases.
So at PubMatic we recommend to execute our tracker call in an iframe tag than image tag.

**Comment by mkendall07** [comment]
Thanks. LGTM.

**Comment by pm-harshad-mane** [comment]
Thank you @mkendall07

**Pull request merged**